src/views/InitSplash.vala

```diff
@@ -0,0 +1,46 @@
+/*-
+ * Copyright (c) 2018-2019 Subhadeep Jasu <subhajasu@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License 
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Authored by: Subhadeep Jasu <subhajasu@gmail.com>
+ */
+
+namespace Hemera.App {
+    public class InitSplash : Gtk.Grid {
+        public signal void user_attempt_reconnect ();
+        public class InitSplash () {
+            var welcome = new Granite.Widgets.Welcome ("Hemera", "Your own personal digital assistant");
+            welcome.append ("media-playback-start","Start Mycroft", "Mycroft is my soul and I need it to function");
+            welcome.append ("application-default-icon", "Setup Mycroft", "Mycroft not installed? Let me help you");
```

Comment:
I  = "Hemera"
Anything in first person refers to the app personified.
Second person words refer to the user.
So, in both the sentences, "I" and "me" refer to Hemera, where as "you" refer to user.

Reply:
Oh, ok. I didn't get the first message then :p

Anyway, I think this isn't really a description because `Mycroft is my soul and I need it to function` seems to fit for both options.

Maybe you could use this text as the subtitle and use something else for the first option? :)

Reply:
That whole part will be redesigned in a similar way to how elementary's first run experience is.